integration-tests/cli/discount-code-importer.it.js

```diff
@@ -0,0 +1,235 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import path from 'path'
+import Promise from 'bluebird'
+import { createAuthMiddlewareForClientCredentialsFlow }
+from '@commercetools/sdk-middleware-auth'
+import { createClient } from '@commercetools/sdk-client'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import { exec } from 'child_process'
+import { getCredentials } from '@commercetools/get-credentials'
+import DiscountCodeImport from '../../packages/discount-code-importer/src/main'
```

Comment:
@junajan No, it actually gets the current version in development as it always builds before the tests. It's all about semantics, and this is what is done in the other ITs. Wouldn't want to break now from convention